WordPressCom-Stats-iOS/StatsTableViewController.m

```diff
@@ -1074,6 +1078,16 @@ - (void)configureTwoColumnRowCell:(UITableViewCell *)cell withLeftText:(NSString
         widthConstraint.constant = 0.0f;
         spaceConstraint.constant = 0.0f;
     }
+    
+    BOOL isNestedRow = indentLevel > 1;
+    if (isNestedRow) {
+        cell.backgroundColor = [WPStyleGuide itsEverywhereGrey];
+    } else {
+        cell.backgroundColor = [UIColor whiteColor];
+    }
+    
+    CGFloat indentWidth = indentLevel * 7.0f;
```

Comment:
This could be a constant but since it's a value defined in the Storyboard, it serves no purpose to define it when it's used in a single spot.